### ISSUE TITLE
Train model with objectron

### ISSUE THREAD

**Issue by bela572** [google/mediapipe#2054]
Where can I find model training(and evaluation) code with objectron dataset?
I would want to use pretrained model for example of shoes and continue training and finding loss

**Comment by ahmadyan** [comment]
Objectron dataset and the evaluation code: https://github.com/google-research-datasets/Objectron/

Pre-trained models: we haven't released the full pre-trained models yet (they will be released around CVPR in the objectron bucket and linked in the objectron repo). The current set are available here: https://github.com/google/mediapipe/tree/master/mediapipe/modules/objectron

Finally, we haven't released the training code.

**Comment by sgowroji** [comment]
Hi @bela572, Have a look at the above [comment](https://github.com/google/mediapipe/issues/2054#issuecomment-844481771)

**Comment by sgowroji** [comment]
Hi @bela572, Could you please respond to my above comment. Thanks!

**Comment by sgowroji** [comment]
Due to lack of activity we are closing this issue now.

**Comment by Vineet-Pandey** [comment]
Hello @ahmadyan - Is there a plan to release the training code in the near future? TIA.
